.cbmc-batch/jobs/aws_cryptosdk_serialize_frame/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags: "--bounds-check;--div-by-zero-check;--float-overflow-check;--nan-check;--pointer-check;--pointer-overflow-check;--signed-overflow-check;--undefined-shift-check;--unsigned-overflow-check;--unwind;1;--unwinding-assertions;--object-bits;8"
+goto: aws_byte_cursor_advance_harness.goto
```

Comment:
This name appears to be wrong, and thus CBMC fails to actually perform any verification.

Reply:
Thanks, I added the `cbmc-batch.yaml` rule in the Makefile.